web/packages/teleport/src/Navigation/Navigation.tsx

```diff
@@ -93,90 +147,236 @@ function getCategoryForRoute(
+          <Section
+            section={section}
+            active={section.category === currentView.category}
+            key={section.category}
+            setExpandedSection={setExpandedSection}
+          />
+        ))}
+      </SideNavContainer>
+      {/* // TODO(rudream): Implement button to make panel sticky.*/}
+      <RightPanel
+        isVisible={!!expandedSection}
+        onMouseLeave={() => setExpandedSection(null)}
+      >
+        <Flex
+          css={`
+            padding-top: ${props => props.theme.space[12]}px;
+            padding-bottom: ${props => props.theme.space[12]}px;
+            padding-left: ${props => props.theme.space[3]}px;
+            padding-right: ${props => props.theme.space[3]}px;
+          `}
+        >
+          <Text typography="h2" color="text.slightlyMuted">
+            {expandedSection?.category}
+          </Text>
+        </Flex>
+        {expandedSection?.subsections.map(section => (
+          <SubsectionItem
+            active={currentView.route === section.route}
+            to={section.route}
+            exact={section.exact}
+            key={section.title}
           >
-            the Downloads page
-          </Text>{' '}
-          for the official release.
-        </>
-      }
-    />
+            <section.icon size={16} />
+            <Text typography="body2">{section.title}</Text>
+          </SubsectionItem>
+        ))}
+      </RightPanel>
+      {/* TODO(rudream): Figure out best place for for license footers.
+        <Flex>
+          {cfg.edition === 'oss' && <AGPLFooter />}
+          {cfg.edition === 'community' && <CommunityFooter />}
+        </Flex> */}
+    </>
   );
 }
 
-function CommunityFooter() {
+const SubsectionItem = styled(NavLink)<{ active: boolean }>`
+  display: flex;
+  position: relative;
+  color: ${props => props.theme.colors.text.slightlyMuted};
+  text-decoration: none;
+  user-select: none;
+  gap: ${props => props.theme.space[2]}px;
+  padding-top: ${props => props.theme.space[12]}px;
+  padding-bottom: ${props => props.theme.space[12]}px;
+  padding-left: ${props => props.theme.space[3]}px;
+  padding-right: ${props => props.theme.space[3]}px;
+
+  ${props => getSubsectionStyles(props.theme, props.active)}
+`;
+
+function Section({
+  section,
+  active,
+  setExpandedSection,
+}: {
+  section: NavigationSection;
+  active: boolean;
+  setExpandedSection: (category: NavigationSection) => void;
+}) {
   return (
-    <LicenseFooter
-      title="Community Edition"
-      subText="Limited Features"
-      infoContent={
-        <>
-          <Text
-            as="a"
-            href="https://goteleport.com/signup/enterprise/?utm_source=oss&utm_medium=in-product&utm_campaign=limited-features"
-            target="_blank"
-          >
-            Upgrade to Teleport Enterprise
-          </Text>{' '}
-          for SSO, just-in-time access requests, Access Graph, and much more!
-        </>
-      }
-    />
+    <CategoryButton
```

Comment:
These are regular divs and not buttons, so they don't work with keyboard navigation OOTB. We should probably consider how to make the side nav accessible through the keyboard alone.